resources/prefs.json

```diff
@@ -1,14 +1,15 @@
 {
   "dom.bluetooth.enabled": false,
   "dom.bluetooth.testing.enabled": false,
-  "dom.canvas-text.enabled": true,
+  "dom.canvas-text.enabled": true, 
   "dom.compositionevent.enabled": false,
   "dom.customelements.enabled": true,
   "dom.forcetouch.enabled": false,
   "dom.gamepad.enabled": false,
   "dom.microdata.testing.enabled": true,
   "dom.mouseevent.which.enabled": false,
   "dom.mutation_observer.enabled": true,
+  "dom.offscreen_canvas.enabled": true,
```

Comment:
We don't actually want to enable this preference by default until the implementation is ready.